media/js/firefox/campaign/unfck.js

```diff
@@ -28,33 +28,39 @@
     }
 
     // FB Share
-    // function shareFacebook(url, text, image) {
-    //     open('http://facebook.com/sharer.php?s=100&p[url]=' + url + '&p[images][0]=' + image + '&p[title]=' + text, 'fbshare', 'height=380,width=660,resizable=0,toolbar=0,menubar=0,status=0,location=0,scrollbars=0');
-    //     // mobile: open('https://m.facebook.com/sharer/sharer.php?u=' + urlLocation + '&picture=' + image + '&description=' + text, 'facebookshare');
-    // }
+    function openFacebookSubwin(url) {
+        open(url, 'fb_share', 'height=380,width=660,resizable=0,toolbar=0,menubar=0,status=0,location=0,scrollbars=0').focus();
+    }
 
     function handleShareLinkClick(e) {
-        var href = e.target.href;
+        var link_href = e.target.href;
+        var service = '';
+
+        if(link_href.includes('twitter')) {
```

Comment:
It looks like `includes` isn't supported by IE, so you may want to change to `indexOf()`?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes